apps/task/models.py

```diff
@@ -99,17 +105,16 @@ def clean_dates(values, instance=None):
                 date=date,
                 user=user,
                 ).exists():
-            errors['date'] = gettext('This time slot overlaps with another '
-                                  'for this day')
+            errors['date'] = gettext('This time entry overlaps with another '
+                                     'for this day')
 
         return errors
 
     @property
     def duration(self):
```

Comment:
Add tests `test_models.py`. For `clean` as well as `property`

if not tested via `test_api`